### PR TITLE
Some updates + small fixes

### PR DESCRIPTION
**Pull request by cyb3rko**
I removed non working links from the README, updated some dependencies, fixed some spelling mistakes :) and removed a redundant identifier.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,16 +15,16 @@ buildscript {
         versions = [
                 androidX        : '1.0.0',
                 cardview        : '1.0.0',
-                appcompat       : '1.1.0',
+                appcompat       : '1.3.0-alpha02',
```
Comment (mikepenz) [inline build.gradle:18]: I think we should stay with stable releases

**Review by mikepenz — CHANGES_REQUESTED** [review]

**Comment by mikepenz** [comment]
Thank you so much for the PR really appreciate. Please see the change request I mentioned

**Comment by cyb3rko** [comment]
Did I miss something that I have to do to allow this PR to get merged or do you just wait for the next release?

**Comment by mikepenz** [comment]
@cyb3rko sorry, thins have been very busy. Yes please see the change here to stay with the stable release for the dependency: https://github.com/mikepenz/AboutLibraries/pull/526/files#diff-c197962302397baf3a4cc36463dce5eaR18

**Pull request merged**